vendor.conf

```diff
@@ -1,9 +1,9 @@
-k8s.io/kubernetes 305052d6d2c1fa976c7a841350396061a2c26ac0 https://github.com/kubernetes/kubernetes
+k8s.io/kubernetes 7fed970f36789786c75ee14f92a38ec3ba746170 https://github.com/kubernetes/kubernetes
 k8s.io/client-go 7cd1d3291b7d9b1e2d54d4b69eb65995eaf8888e https://github.com/kubernetes/client-go
```

Comment:
don't we need these other deps to be bumped as well? the timeout might be some misconfig in libs here (utils, client-go)